src/mistral_common/protocol/instruct/messages.py

```diff
@@ -161,28 +164,64 @@ class RawAudio(MistralBase):
     format: str
 
     @classmethod
-    def from_audio(cls, audio: Audio) -> "RawAudio":
+    def from_audio(cls, audio: Audio, prefix: bool = False) -> "RawAudio":
```

Comment:
Hmm I don't think we should add a prefix here

Reply:
I don't see a use case where we'd want to have a prefix as part of the "raw" audio